pkg/actions/lua/storage/aws/glue.go

```diff
@@ -223,3 +225,40 @@ func getTable(c *GlueClient) lua.Function {
 		return 2
 	}
 }
+
+func createDatabase(c *GlueClient) lua.Function {
+	return func(l *lua.State) int {
+		client := c.client()
+		database := lua.CheckString(l, 1)
+		description := "Created by lakeFS Action"
+		if !l.IsNone(2) {
+			description = lua.CheckString(l, 2)
+		} // AWS API call
+		_, err := client.CreateDatabase(c.ctx, &glue.CreateDatabaseInput{
```

Comment:
For create database input it's important to enable the different options otherwise we will be busy chasing specific params. 
See this [input](https://docs.aws.amazon.com/glue/latest/webapi/API_CreateDatabase.html) 
We should allow passing w/e the user needs, while setting reasonable defaults

Reply:
OK, I asked @talSofer for her input what params are important here

Reply:
> OK, I asked @talSofer for her input what params are important here

Sure, that's great input, but it's first of all for R&D to not being blocker. 
This is a low level SDK that shouldn't be opinionated about this. 
As a design principal, the lowest common point should allow the user choice. 
Take a look at [glue.create_table](https://github.com/treeverse/lakeFS/blob/master/pkg/actions/lua/storage/aws/glue.go#L152) it takes `Create Table Input` because there are different configuration parameters that can be used.

In this case for example - if someone wants to create the DB with certain permissions or a federated one (all parts of the input to create db) - it's perfectly fine from lakeFS side. They should be able to do it without creating an issue for lakeFS, waiting for release and then upgrading lakeFS Server.